report_thesis/src/sections/background.tex

```diff
@@ -42,6 +42,38 @@ \subsubsection{Norm 3}
 This normalization method results in a total of $3N = 6144$ normalized features for each sample, as each of the three spectrometers contributes 2048 channels.
 
 \subsubsection{Power Transformation}
+Power transformations are a class of mathematical functions used to stabilize variance and make data more closely approximate a normal distribution. 
+They are particularly useful in statistical modeling and data analysis to meet the assumptions of linear models.
+
+One of the first influential power transformation techniques is the Box-Cox power transform, introduced by \citet{BoxAndCox} in 1964.
+This is defined for positive data and is aimed at normalizing data or making it more symmetric. The transformation is given by:
+
+$$ 
+\text{BC}(\lambda, x) = 
+\begin{cases} 
+\frac{x^\lambda - 1}{\lambda} & \text{if } \lambda \neq 0 \\
+\log(x) & \text{if } \lambda = 0 
+\end{cases} 
+$$
+where $ \lambda $ is the transformation parameter and $x$ is the input data. 
```

Comment:
I think this might needs a bit more information about what lambda is? If they ask us at the exam, I wouldn't be able to tell you based on this definition.